etc/profile-a-l/gtk-pipe-viewer.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for gtk-pipe-viewer
+# Description: Gtk front-end to pipe-viewer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gtk-pipe-viewer.local
+# added by included profile
+#include globals.local
+
+ignore quiet
+
+include whitelist-runuser-common.inc
```

Comment:
Why only here?

IDK how pipe-viewer works but it's maybe work with `blacklist ${RUNUSER}`, then we could add it and a `noblacklist ${RUNUSER}` here.